src/utils/components/CloudinitModal/utils/cloudinit-utils.ts

```diff
@@ -102,9 +107,20 @@ export const createDefaultCloudInitYAML = () => ({
   userData: '',
 });
 
+export const addCloudInitUpdateCMD = (userData: CloudInitUserData) => {
+  userData.packages
+    ? userData.packages.push(DNF_AUTOMATIC_PACKAGE)
+    : (userData.packages = [DNF_AUTOMATIC_PACKAGE]);
```

Comment:
I was thinking, maybe the following could work, too?
```
userData.packages ??= [];
userData.packages.push(DNF_AUTOMATIC_PACKAGE);
```
Simpler and we could save a line of the code, too. WDYT?